include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/PositionDerivative.hpp

```diff
@@ -60,6 +60,12 @@ class PositionDerivative : public Dynamics
 
     virtual bool isDefined() const override;
 
+    /// @brief              Declares the coordinates involved in the contribution calculation
+    ///
+    /// @param              [in] coordinatesBroker The coordinate broker mapping state coordinates and indexes
+
+    virtual void declareCoordinates(const Shared<CoordinatesBroker>& coordinatesBroker) override;
```

Comment:
```suggestion
    virtual void declareCoordinates(const Shared<CoordinatesBroker>& coordinatesBroker) const override;
```

Reply:
Cannot make it const as it might modify the instance (for instance, storing internal indexes, see `AtmosphericDrag` example implementation)